docs/interfaces.md

```diff
@@ -379,3 +379,9 @@ Can manage timeservers directly separate from ``config core``.
 Can access the tpm device /dev/tpm0.
 
 * Auto-Connect: no
+
+### fwupd
+
+Allow fwupd to update UEFI capsule format firmware.
```

Comment:
The name of this interface doesn't quite fit with other interface names. Also, this should be in alphabetical order in interfaces.md. Please don't change the name of this interface yet, but what do people think of 'uefi-firmware-control' instead? @niemeyer, @zyga?